topydo/ui/columns/Main.py

```diff
@@ -283,27 +301,31 @@ def _execute_handler(self, p_command, p_todo_id=None, p_output=None):
                 'Error: {}. Check your aliases configuration.'.format(cerr))
             return
 
+        self._backup(subcommand, args)
+
         env_args = (self.todolist, p_output, self._output, self._input)
 
         if '{}' in args:
-            ids = self.marked_todos if self._has_marked_todos() else [p_todo_id]
+            if self._has_marked_todos():
+                ids = self.marked_todos
+            else:
+                ids = set(p_todo_id) if p_todo_id else set()
```

Comment:
This line will convert p_todo_id (which is a string) into a set of single characters. For example `'rlr'` will become `{'r', 'l'}`. I will fix that in next push.